.appveyor.yml

```diff
@@ -0,0 +1,18 @@
+build: false
+
+environment:
+  matrix:
+    - MINICONDA: "C:\\Miniconda36-x64"
+      PYTHON: 3.6
```

Comment:
3.6 here, but on circle I see a 3.8 environment is being created. Which version(s) do we intend to support?

This discrepancy can lead to issues, since some pip packages cannot be installed on windows without binary wheels and distributors don't necessarily make such wheels for older package versions and newer python versions.

Reply:
I assumed we would support 3.6+, but I'm open on that